From 8cd2d114635e3bd3a06835b908a1289af2ad0e5b Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Thu, 6 Aug 2015 11:27:40 -0400
Subject: [PATCH 355/363] Ticket 48215  - update dbverify usage in main.c

Description:  Need to update dbverify usage in main.c

https://fedorahosted.org/389/ticket/48215
(cherry picked from commit c1912cdcac8319e2fe0f98f765aa935e6a8ff297)
(cherry picked from commit aa083988ed65bf9ae9d0f4090ce3ef4ce5cb4a31)
---
 ldap/servers/slapd/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/servers/slapd/main.c b/ldap/servers/slapd/main.c
index 2442610..95976d3 100644
--- a/ldap/servers/slapd/main.c
+++ b/ldap/servers/slapd/main.c
@@ -453,7 +453,7 @@ usage( char *name, char *extraname )
 	usagestr = "usage: %s %s%s-D configdir [-d debuglevel] [-N] -n backend-instance-name -a fullpath-backend-instance-dir-full\n";
 	break;
     case SLAPD_EXEMODE_DBVERIFY:
-	usagestr = "usage: %s %s%s-D configdir [-d debuglevel] [-n backend-instance-name]\n";
+	usagestr = "usage: %s %s%s-D configdir [-d debuglevel] [-n backend-instance-name] [-a db-directory]\n";
 	break;
 
     default:	/* SLAPD_EXEMODE_SLAPD */
-- 
2.4.3

